packages/forms-web-app/src/dynamic-forms/middleware/check-not-submitted.js

```diff
@@ -10,9 +8,18 @@ const {
 module.exports = (alreadySubmittedUrl) => (req, res, next) => {
 	const journeyResponse = res?.locals?.journeyResponse;
 
+	let userAppealOverview;
+	if (req.session?.user?.isLpaUser) {
+		userAppealOverview = LPA_DASHBOARD.APPEAL_OVERVIEW;
+	} else if (req.session?.user?.isRule6User) {
+		userAppealOverview = RULE_6.APPEAL_OVERVIEW;
+	} else {
+		userAppealOverview = SELECTED_APPEAL.APPEAL_OVERVIEW;
+	}
```

Comment:
aren't you passing the view into this middleware as the param when it is called in the router? not sure I understand why it is set here - unless you need to check the user type as well?